File: mobile/src/main/java/com/windscribe/mobile/welcome/fragment/WelcomeActivityCallback.java
/*
 * Copyright (c) 2021 Windscribe Limited.
 */

package com.windscribe.mobile.welcome.fragment;


public interface WelcomeActivityCallback {

    void clearInputErrors();

    void setLoginError(String error);

    void setPasswordError(String error);

    void setUsernameError(String error);

}

File: mobile/src/main/java/com/windscribe/mobile/welcome/fragment/WelcomeActivityCallback.kt
/*
 * Copyright (c) 2021 Windscribe Limited.
 */
package com.windscribe.mobile.welcome.fragment

interface WelcomeActivityCallback {
    fun clearInputErrors()
    fun setLoginError(error: String)
    fun setPasswordError(error: String)
    fun setUsernameError(error: String)
}